schema-util/json/src/test/java/io/apicurio/registry/rules/compatibility/jsonschema/JsonSchemaSmokeTest.java

```diff
@@ -94,6 +94,17 @@ public void testComparison() throws Exception {
                             caseId, backwardCompatible, backward.getDiff(), forwardCompatible, forward.getDiff());
                     }
                     break;
+                case "forward":
```

Comment:
No need to add the forward case, because you can reverse the schema in the tests file (forward compatibility is just reversed backward compatibility)